### DISCUSSION TITLE
21 - Modern Web Frontend

### DISCUSSION THREAD

**Discussion by kevinslin** [dendronhq/dendron#1145] [RFCs/Ideas]
Discussion thread for Dendron's [Modern Web Frontend](https://wiki.dendron.so/notes/nB75cmdPhL7CTOATG6wpT.html)

**Comment by xypnox** [comment]
- The sidebar tree is not scrollable.

**Reply by kevinslin** [reply]
apologies for the long delay. this is now fixed in the latest version of the template (see https://blog.dendron.so/)

**Comment by neeraj9** [comment]
Did you look at using other programming language apart from NodeJS? There are numerous projects (lets say Hugo), which speeds up the static site generation a lot via non-JS programming languages.

**Reply by xypnox** [reply]
Apart from Hugo I really like the no nonsense generation of zola. https://www.getzola.org/

**Reply by kevinslin** [reply]
we have for inspiration but ended up coming back to nextjs because it builds on top of our existing toolchain (typescript, express, react, etc) and lets us re-use components between publishing and the dendron client
